docs/src/man/getting_started.md

```diff
@@ -476,6 +476,90 @@ julia> df[in.(df.A, Ref([1, 5, 601])), :]
 
 Equivalently, the `in` function can be called with a single argument to create a function object that tests whether each value belongs to the subset (partial application of `in`): `df[in([1, 5, 601]).(df.A), :]`.
 
+#### Column selection using `select` and `select!`
+
+You can also use the [`select`](@ref) and [`select!`](@ref) functions to select columns in a data frame.
+
+The `select` function creates a new data frame:
+```jldoctest dataframe
+julia> df = DataFrame(x1=1, x2=2, y=3)
+1×3 DataFrame
+│ Row │ x1    │ x2    │ y     │
+│     │ Int64 │ Int64 │ Int64 │
+├─────┼───────┼───────┼───────┤
+│ 1   │ 1     │ 2     │ 3     │
+
+julia> select(df, Not(:x1)) # drop column :x1 in a new data frame
+1×2 DataFrame
+│ Row │ x2    │ y     │
+│     │ Int64 │ Int64 │
+├─────┼───────┼───────┤
+│ 1   │ 2     │ 3     │
+
+julia> select(df, r"x") # select columns containing 'x' character
+1×2 DataFrame
+│ Row │ x1    │ x2    │
+│     │ Int64 │ Int64 │
+├─────┼───────┼───────┤
+│ 1   │ 1     │ 2     │
+```
+
+It is important to note that `select` always returns a data frame,
+even if a single column is selected (as opposed to indexing syntax).
+```jldoctest dataframe
+julia> select(df, :x1)
+1×1 DataFrame
+│ Row │ x1    │
+│     │ Int64 │
+├─────┼───────┤
+│ 1   │ 1     │
+
+julia> df[:, :x1]
+1-element Array{Int64,1}:
+ 1
+```
+
+By default `select` copies columns of a passed source data frame.
+In order to avoid copying set its `copycols` keyword argument to `false`:
+```
+julia> df2 = select(df, :x1)
+1×1 DataFrame
+│ Row │ x1    │
+│     │ Int64 │
+├─────┼───────┤
+│ 1   │ 1     │
+
+julia> df2.x1 === df.x1
+false
+
+julia> df2 = select(df, :x1, copycols=false)
+1×1 DataFrame
+│ Row │ x1    │
+│     │ Int64 │
+├─────┼───────┤
+│ 1   │ 1     │
+
+julia> df2.x1 === df.x1
+true
+```
+
+To perform the selection operation in-place use `select!`:
+```jldoctest dataframe
+julia> select!(df, Not(:x1))
```

Comment:
I'd show `df` right after this, instead of the result of `select!`. Then you can drop the second example.

Reply:
fixed - is this what you had in mind?